helpers/calendar/analytics_helper.py

```diff
@@ -38,6 +38,23 @@ def convert_dates(self, start_date, end_date):
         end_date = (datetime.strptime(end_date, "%b %d %Y") + relativedelta(days=1)).isoformat() + 'Z'  # noqa: E501
         return(start_date, end_date)
 
+    def validate_current_date(self, start_date, end_date):
+        """
+        Checks for today's date
+        """
+        start_date, end_date = CommonAnalytics.convert_dates(
+            self, start_date, end_date)
+
+        now = datetime.now().strftime('%b %d %Y')
+        date_now = (datetime.strptime(now, "%b %d %Y") + relativedelta(days=1)).isoformat() + 'Z'  # noqa: E501
```

Comment:
tou can remove the noqa here by doing this:
```
date_now = (datetime.strptime(now, "%b %d %Y") + relativedelta(
            days=1)).isoformat() + 'Z'
```

Reply:
Thanks @kampkelly